### PR TITLE
hints: target="up" or "down"

### PR DESCRIPTION
**Pull request by moyogo**
See *Hints: handle up and down targets* https://github.com/googlefonts/glyphsLib/issues/376

### CODE REVIEW

**Comment by moyogo** [comment]
@madig do we need to check target="down" or target="up" only occurs with type="TopGhost" or "BottomGhost" respectively?

**Comment by madig** [comment]
No idea. @schriftgestalt ?

**Comment by schriftgestalt** [comment]
> target="down" or target="up" only occurs with type="TopGhost" or "BottomGhost"

This is true. Is this answering the question?

**Comment by moyogo** [comment]
>>    target="down" or target="up" only occurs with type="TopGhost" or "BottomGhost"

> This is true. Is this answering the question?

Yes, thanks! I guess glyphsLib doesn’t need to validate.

File: Lib/glyphsLib/builder/hints.py [hunk 1]
```diff
@@ -49,8 +50,14 @@ def to_glyphs_hints(self, ufo_glyph, layer):
         for attr in ["horizontal", "options", "stem", "type"]:
             setattr(hi, attr, hint[attr])
         for attr in ["origin", "other1", "other2", "place", "scale", "target"]:
-            # FIXME: (jany) what about target = up/down?
             if attr in hint:
-                value = Point(*hint[attr])
+                # handle target = ['u', 'p'] or ['d', 'o', 'w', 'n']
```
Comment (anthrotype) [inline Lib/glyphsLib/builder/hints.py:54]: @moyogo why do we need to `handle target = ['u', 'p'] or ['d', 'o', 'w', 'n']` ? Is that for backward compat with how these attribute was written out to UFO before this PR? If so, maybe add a comment with a link to the issue

**Review by anthrotype — COMMENTED** [review]
What's the status of this?

BTW, I think the flake8 errors should pass after rebasing or merging master onto this.

**Review by anthrotype — APPROVED** [review]

**Pull request merged**